### ISSUE TITLE
404 error

### ISSUE THREAD

**Issue by Sam948-byte** [3kh0/discord-proxy#6]
When I open it I just get this error:
![Screenshot 2021-11-29 17 16 31](https://user-images.githubusercontent.com/66854780/143957895-39a07f9d-fee6-4be1-b936-369590e7bab5.png)

**Comment by 3kh0** [comment]
Yes, right now this does not work very well. I suggest you use other proxies. If you need help then please reply. :)

**Comment by Sam948-byte** [comment]
I've done some digging, and so far I haven't found a good proxy that supports discord.  Do you have any recommandations?

**Comment by BuggyThe1st** [comment]
no, not really

**Comment by 3kh0** [comment]
This proxy is based on holy unblocker and we are exercising issues at the moment.
